ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -636,7 +663,7 @@ private enum UnityLayers {
 
     if (hide_all_gui_) {
```

Comment:
`hide_all_gui_ || !in_principia_scene_`, so that it's clearer that we treat those equivalently.

Reply:
Done.